src/org/zaproxy/zap/extension/pscanrulesBeta/CSRFCountermeasures.java

```diff
@@ -148,6 +147,7 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 			
 			for (Element formElement : formElements) {
 				logger.debug("FORM ["+ formElement + "] has parent ["+ formElement.getParentElement()+"]");
+				boolean foundCsrfToken = false;
```

Comment:
This can be declared even later (assuming the statement at line 157 is removed, it no longer has effect).